src/connections/DelayedJob.ts

```diff
@@ -0,0 +1,103 @@
+import { useEffect } from "react";
+import { createSelector } from "reselect";
+
+import { bulkUpdateJobs, listDelayedJobs } from "@/generated/v3/jobService/jobServiceComponents";
+import {
+  bulkUpdateJobsFetchFailed,
+  bulkUpdateJobsIsFetching,
+  listDelayedJobsFetchFailed
+} from "@/generated/v3/jobService/jobServicePredicates";
+import { DelayedJobData, DelayedJobDto } from "@/generated/v3/jobService/jobServiceSchemas";
+import { ApiDataStore } from "@/store/apiSlice";
+import { Connection } from "@/types/connection";
+import { connectionHook, connectionLoader } from "@/utils/connectionShortcuts";
+
+// --- Delayed Jobs Connection ---
+type DelayedJobsConnection = {
+  delayedJobs?: DelayedJobDto[];
+  isLoading: boolean;
+  hasLoadFailed: boolean;
+};
+
+const delayedJobsSelector = (store: ApiDataStore) => {
+  const delayedJobsMap = store.delayedJobs || {};
```

Comment:
Please use ?? for defaults. I'd reformat this whole method:
```
const delayedJobsSelector = (store: ApiDataStore) => 
  Object.values(store.delayedJobs ?? {}).map(resource => resource.attributes);
```

Reply:
changed